### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by eXtreme-by** [exaile/exaile#643]
> Traceback (most recent call last):
  File "/usr/local/lib/exaile/xlgui/playlist_container.py", line 83, in do_clicked
    self.notebook.create_new_playlist()
  File "/usr/local/lib/exaile/xlgui/playlist_container.py", line 206, in create_new_playlist
    name = page.get_page_name().decode('utf-8', errors='replace')
  File "/usr/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-5: ordinal not in range(128)

Distributor ID:	LinuxMint
Description:	Linux Mint 19.1 Tessa x64

**Comment by rokm** [comment]
I can reproduce this with following steps (Fedora 30, exaile git master):
1. Create a playlist
2. Rename the playlist to a name that contains a non-ASCII character (e.g., "Playlist with Š")
3. Save the playlist (using the name from Step 2)
4. Close & restart
5. Restore the saved playlist from Playlists -> Custom Playlists
6. Try to add a new playlist

This results in the same traceback as in the original report.

I suspect the saving and restoring part may be necessary in order for the playlist name to be set as page name.

**Comment by rokm** [comment]
A related issue:
1. Create a playlist
2. Rename the playlist to a name that contains a non-ASCII character (e.g., "Playlist with Š")
3. Close exaile

This results in the following error:
```
INFO    : Saving state...
ERROR   : Error saving tab 'order1.current.Playlist with \xc5\xa0'
Traceback (most recent call last):
  File "/home/rok/Projects/exaile/xlgui/playlist_container.py", line 314, in save_current_tabs
    self.tab_manager.save_playlist(page.playlist, True)
  File "/home/rok/Projects/exaile/xl/playlist.py", line 2119, in save_playlist
    pl.save_to_location(os.path.join(self.playlist_dir, encode_filename(name)))
  File "/usr/lib64/python2.7/posixpath.py", line 73, in join
    path += '/' + b
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc5 in position 30: ordinal not in range(128)
INFO    : Bye!
```

**Comment by virtuald** [comment]
Fixed by #644
